.github/ISSUE_TEMPLATE/2-feature-request.md

```diff
@@ -0,0 +1,15 @@
+---
+name: "⭐ Feature request"
+about: Suggest an idea for Got
+
+---
+
+#### What's the idea?
```

Comment:
The first question should be:

```
### What problem are you trying to solve?
```

Many times, people post ideas that sound like a bad idea, but the actual problem is valid, just needs a better solution. And sometimes the problem is already solved in a better way. That's why knowing the problem to solve is more important than the suggested idea.